README.md

```diff
@@ -1,4 +1,114 @@
-# cluster-api-provider-baiducloud
+# Cluster Api Provider Baiducloud
 
-## Documentations
-Currently you can get all documentations under the `/docs` directory.
+## Components
+
+### Binaries
+
+* cluster-api-controller: The one which has no dependencies with baiducloud provider.
+* cluster-controller: baiducloud cluster controller
+* machine-controller: baiducloud machine controller
```

Comment:
only one controller now, responsible for both cluster and machine

Reply:
Outdated...